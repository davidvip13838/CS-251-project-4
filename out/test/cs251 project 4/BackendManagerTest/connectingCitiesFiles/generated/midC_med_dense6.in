500
48 86
70 31
27 65
15 27
2 55
76 36
5 24
26 3
33 95
73 71
23 76
64 19
33 67
37 2
22 56
86 88
46 43
94 75
5 80
5 65
90 32
71 67
67 63
41 96
41 80
96 16
24 38
40 66
88 64
11 0
44 31
9 46
97 37
42 29
16 100
78 84
16 77
18 39
50 97
87 60
14 2
74 0
5 67
32 14
96 64
2 2
32 98
85 33
93 62
57 45
86 46
78 78
85 37
9 98
86 80
24 93
17 37
11 76
63 16
27 57
95 90
40 38
63 29
45 24
15 24
29 25
82 83
16 56
16 98
8 44
76 52
58 27
24 22
60 33
81 70
47 47
81 17
70 55
13 95
79 32
90 10
70 21
89 15
65 29
39 89
38 74
87 49
53 19
21 53
4 11
99 77
10 62
25 50
82 40
99 49
7 11
69 77
6 44
2 93
32 22
28 63
95 67
15 26
34 36
55 33
100 33
26 45
88 14
13 63
59 95
6 0
86 28
56 29
67 80
1 37
82 9
39 66
80 42
38 40
83 8
90 7
11 11
95 1
92 80
47 1
31 27
13 36
49 21
21 46
43 72
95 47
17 60
19 59
29 47
85 74
100 29
10 83
52 56
68 16
71 79
87 17
23 23
80 54
2 30
53 71
86 33
80 89
63 86
91 100
94 15
95 64
69 15
69 27
54 39
85 100
77 3
100 11
17 79
73 94
66 65
48 90
88 9
81 62
83 100
27 4
75 98
8 42
94 85
56 4
17 35
60 69
63 34
7 2
93 22
64 10
10 42
23 83
32 40
13 73
81 25
90 42
56 78
57 19
16 23
64 77
29 49
73 11
31 32
100 70
2 95
85 96
75 61
58 19
32 47
79 85
4 99
45 62
35 25
1 29
50 89
86 29
40 12
84 60
93 26
15 0
30 76
6 13
73 88
11 54
27 95
16 47
43 70
54 5
2 58
27 58
52 64
30 12
61 31
97 95
45 60
73 95
63 15
31 11
38 2
45 65
100 31
96 30
75 50
9 100
100 27
36 12
47 19
94 30
24 20
83 69
22 23
96 19
9 9
99 91
92 88
58 64
34 50
68 18
65 55
7 64
84 84
28 57
79 80
67 87
76 17
36 95
28 66
51 83
65 4
13 89
71 32
44 68
44 57
40 65
62 43
87 70
85 50
24 85
55 21
62 16
19 22
97 80
52 92
5 11
18 16
38 55
33 24
5 59
25 51
67 48
26 68
77 80
68 79
36 85
92 24
65 18
5 53
47 63
7 32
63 31
11 82
94 51
63 3
25 7
54 87
52 33
82 80
64 50
78 10
52 53
83 11
41 56
6 17
38 87
88 41
34 38
19 11
94 81
2 92
55 77
48 63
63 37
99 10
49 27
16 91
14 18
30 98
5 4
41 5
40 46
21 47
69 3
81 97
91 91
12 42
48 41
65 15
19 92
60 52
68 28
20 22
22 61
48 19
79 7
100 5
34 16
54 9
16 8
6 81
13 11
35 50
7 54
58 54
83 18
7 7
23 65
54 58
19 2
98 51
69 28
6 36
37 9
16 41
17 78
87 73
25 70
25 41
58 42
8 13
34 85
82 26
91 4
91 79
94 68
3 44
64 24
27 14
9 56
20 67
25 28
9 44
28 80
43 89
79 100
54 52
71 3
1 81
94 19
97 38
37 89
31 63
18 76
8 66
16 92
3 8
77 91
98 64
12 72
49 53
32 87
28 83
93 29
33 8
51 48
82 91
100 3
12 7
55 52
1 27
0 71
73 35
9 41
21 28
0 76
2 5
48 4
47 75
22 39
79 44
21 12
38 85
15 98
49 68
86 89
12 56
98 56
65 42
71 90
66 53
85 14
21 76
56 60
1 4
1 46
81 86
21 61
33 38
34 22
1 0
98 96
19 10
99 13
49 100
65 26
25 12
98 18
31 58
92 37
44 15
80 21
5 18
48 1
76 59
39 37
28 37
54 100
9 34
79 88
67 89
44 90
39 39
66 48
34 88
39 61
46 39
58 5
8 93
60 41
26 30
18 80
64 42
0 39
48 16
27 48
23 20
18 38
27 62
44 21
34 32
63 68
73 97
76 7
36 62
30 35
72 24
34 6
20 88
87 55
57 9
23 29
88 72
75 94
19 17
94 7
31 24
45 80
25 42
38 32
16 73
31 65
17 72
94 28
77 77
12 75
4 94
22 70
6 24
26 48
91 95
62 25
24 77
43 13
64 28
65 91
66 75
