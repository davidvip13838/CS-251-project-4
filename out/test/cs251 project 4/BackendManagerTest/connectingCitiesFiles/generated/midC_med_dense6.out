1528.88601476935